modules/openxr/openxr_api.h

```diff
@@ -104,8 +104,10 @@ class OpenXRAPI {
 
 	// blend mode
 	XrEnvironmentBlendMode environment_blend_mode = XR_ENVIRONMENT_BLEND_MODE_OPAQUE;
+	XrEnvironmentBlendMode requested_environment_blend_mode = XR_ENVIRONMENT_BLEND_MODE_OPAQUE;
```

Comment:
I wonder if we should start using a `struct` variable instead for this pattern (we used the same approach for the `reference_space`) to ensure that updates to this logic does not inadvertently forget to account for the other field.

```
struct OpenXRBlendMode {
    XrEnvironmentBlendMode environment_blend_mode = XR_ENVIRONMENT_BLEND_MODE_OPAQUE;
    XrEnvironmentBlendMode requested_environment_blend_mode = XR_ENVIRONMENT_BLEND_MODE_OPAQUE;
    bool emulate_environment_blend_mode_alpha_blend = false;
}

OpenXRBlendMode blend_mode;
```

We should also document the differences between the two values (should also do the same for `reference_space`) to make it clear how they are being used.

Reply:
I like this, all for clearly grouping things that belong together.

Reply:
Are you envisioning this including these two as well?

```c++
	uint32_t num_supported_environment_blend_modes = 0;
	XrEnvironmentBlendMode *supported_environment_blend_modes = nullptr;
```

Part of the reason I ask is that `emulate_environment_blend_mode_alpha_blend` field is more a part of the list of supported modes, rather than the current/requested mode.

Or, maybe we should have two structs, like:

```c++
struct OpenXRBlendMode {
    XrEnvironmentBlendMode current_mode = XR_ENVIRONMENT_BLEND_MODE_OPAQUE;
    XrEnvironmentBlendMode requested_mode = XR_ENVIRONMENT_BLEND_MODE_OPAQUE;
};
OpenXRBlendMode blend_mode;

struct OpenXRBlendModeList {
    uint32_t num_modes = 0;
    XrEnvironmentBlendMode *modes = nullptr;
    bool emulate_alpha_blend_mode = false;
};
OpenXRBlendModeList supported_blend_modes;
```

That would also fit the pattern used for reference spaces pretty well.

In any case, either of these refactors will end up changing a whole bunch of code unrelated to passthrough. Could we do that as part of a dedicated PR?

Reply:
The two structs approach looks better. I'd imagine that `OpenXRBlendModeList supported_blend_modes` is initialized once on startup?

And yeah, totally fine to do it in a separate PR.